src/modules/addimage.js

```diff
@@ -975,44 +975,38 @@ import { atob, btoa } from "../libs/AtobBtoa.js";
    * @param {Object} imageData
    * @returns {Object}
    */
-  jsPDFAPI.getImageProperties = function(imageData) {
-    var image;
-    var tmpImageData = "";
-    var format;
+  jsPDFAPI.getImageProperties = function(data) {
+    let imageData;
 
-    if (isDOMElement(imageData)) {
-      imageData = getImageDataFromElement(imageData);
+    if (isDOMElement(data)) {
+      imageData = getImageDataFromElement(data);
     }
 
-    if (
-      typeof imageData === "string" &&
-      getImageFileTypeByImageData(imageData) === UNKNOWN
-    ) {
-      tmpImageData = convertBase64ToBinaryString(imageData, false);
+    // if data is a string, then it should be treated as a URL
+    if (typeof data === "string") {
+      imageData = jsPDFAPI.loadFile(data, true);
+    }
 
-      if (tmpImageData === "") {
```

Comment:
The way that I read this, I interpret this check to be seeing if `imageData` was originally a URL, as that would cause this variable to still be "". I've reordered and split up these operations for clarity. The important thing here is that only _URLs_ should be treated as URLs, _not_ Data URLs.